webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -35,7 +49,39 @@ export default {
   props: {
     placement: { type: String },
   },
+  created() {
+    setInterval(this.updateNotifications, REFRESH_MILLISEC)
+  },
+  destroyed() {
+    clearInterval(this.updateNotifications)
+  },
   methods: {
+    async updateNotifications() {
+      try {
+        const {
+          data: { notifications },
+        } = await this.$apollo.mutate({
+          mutation: notificationQuery(this.$i18n),
```

Comment:
Why is it called `notificationQuery` if it's a mutation? :thinking:

Reply:
You are right @roschaefer ! It is no mutation, but a query!
This is a copy and paste left over. 😱